examples/api/lib/widgets/scrollbar/raw_scrollbar.0.dart

```diff
@@ -34,6 +34,12 @@ class RawScrollbarExample extends StatefulWidget {
 class _RawScrollbarExampleState extends State<RawScrollbarExample> {
   final ScrollController _firstController = ScrollController();
 
+  @override
+  void dispose() {
+    _firstController.dispose();
```

Comment:
Good catch for those missing dispose :tada: 

Consider renaming the `_firstController` variable to `_controller` or `_scrollController` and do the same for the other scrollbar examples.

This one is also wrongly formatted, consider fixing it, as you want.